Crashlytics/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- [changed] Added a check before casting unsigned to signed integer
```

Comment:
I might suggest combining the changelog entry for these 3 PRs just so we aren't highlighting the changes too much individually. Maybe something like, "Incorporated code quality changes around integer overflow and reinstalling signal handlers".